notes/src/components/Modal.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { deleteNote } from '../actions';
+import './Modal.css';
+
+class Modal extends Component {
+
+  cancelModal = () => {
+    const modal = document.getElementsByClassName('modalContainer')[0];
```

Comment:
`document.querySelector('.modalContainer')` would work better in this case.

Reply:
That one is much better to look at. I was a little depressed when I realized there was no getElementByClassName (though the reason is obvious to me now) but you wouldn't believe how happy I was to realize I could pull it out of the collection like that!